scout/server/blueprints/cases/templates/cases/gene_variants.html

```diff
@@ -0,0 +1,195 @@
+{% extends "layout.html" %}
+{% from "variants/components.html" import gene_cell, frequency_cell %}
+{% from "utils.html" import flash_messages, comments_table %}
+
+{% block title %}
+  {{ super() }} - {{ institute.display_name }} - All variants
```

Comment:
yeah, that would be nicer! Me fix!

Reply:
At one point, there will be one place to change those. Perhaps.

Reply:
Hi, now it works and maintains all the tabs while you move from cases to causative and so on. 
I's a small thing but I noticed that in the page "All SNVs and INDELs" there is no space between the main table and the page borders: 
![image](https://user-images.githubusercontent.com/28093618/53720381-eae26980-3e60-11e9-975a-69853bd10c00.png)

Could you encase the table in a container so that it has the same space from the border as the filter panel above?

Reply:
Gotcha! It was the same as among the variant pages, but since I insist keeping it under cases.. 👍